code/modules/cargo/supplypod.dm

```diff
@@ -598,7 +598,7 @@
 	if(pod.effectStun) //If effectStun is true, stun any mobs caught on this pod_landingzone until the pod gets a chance to hit them
 		for (var/mob/living/target_living in get_turf(src))
 			target_living.Stun(pod.delays[POD_TRANSIT]+10, ignore_canstun = TRUE)//you ain't goin nowhere, kid.
-	if (pod.delays[POD_FALLING] == initial(pod.delays[POD_FALLING]) && pod.delays[POD_TRANSIT] + pod.delays[POD_FALLING] < pod.fallingSoundLength)
+	if (pod.delays[POD_TRANSIT] + pod.delays[POD_FALLING] < pod.fallingSoundLength)
```

Comment:
`initial()` on a list index (except for the `vars` list) returns the current value. This is equivalent to `thing == thing`.

Reply:
hilarious

Reply:
So the check for if we're using the same delays never worked?
Shouldn't a better check be added then?

Reply:
@LemonInTheDark Such as?

Reply:
In retrospect I'm not sure why this was even done